clients/input_copy.go

```diff
@@ -63,8 +70,62 @@ func (s *InputCopy) CopyInputToS3(requestID string, inputFile *url.URL) (inputVi
 		}
 	}
 
+	if encryptedKey != "" {
+		c, e := getFile(context.Background(), requestID, inputFile.String())
+
+		if e != nil {
+			glog.Errorf("error getting file: %w", err)
+			return
+		}
+
+		var decryptionKey *rsa.PrivateKey
+		if decryptionKey, err = crypto.LoadPrivateKey(VodDecryptPrivateKey); err != nil {
```

Comment:
nit: How expensive is this loading of the private key? We have it a static config, so ideally should only need to load it once right?